src/iam_roles/iamrolesservicemapping.py

```diff
@@ -164,4 +164,4 @@ def lambda_handler(event, context):
                 "body": "Error invoking iamroleservice_lambda",
             }
 
-    return {"statusCode": 200, "body": json.dumps(resource_mapping)}
+    return {"statusCode": 200, "body": json.dumps(resource_mapping)}
```

Comment:
Add EOF

Reply:
added in d4d700c790b5926a49486872d2b4441380703fa1